packages/components/src/templates/input-with-currency-stepper.stories.ts

```diff
@@ -0,0 +1,62 @@
+import '../solid-components';
+import { html } from 'lit-html';
+import type SdInput from '../components/input/input';
+
+/**
+ * ```
+ * ```
+ */
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Input with Currency Stepper',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+export const Default = {
```

Comment:
Use the "input with currency stepper" title here.

In case we need to add more templates for input in the future we can simply add it here.